app/src/main/java/org/wikipedia/activity/BaseActivity.java

```diff
@@ -110,6 +110,8 @@
         setNavigationBarColor(ResourceUtil.getThemedColor(this, R.attr.paper_color));
 
         maybeShowLoggedOutInBackgroundDialog();
+
+        WikipediaApp.getInstance().putCrashReportProperty("app_local_class_name", getLocalClassName());
```

Comment:
I don't think this will actually work:  The crash report properties are stored in memory. When a crash happens, the properties are wiped.  That's why the only properties that get sent are the ones that are initialized when `WikipediaApp` is first created.